episodes/severity-static.Rmd

```diff
@@ -0,0 +1,641 @@
+---
+title: 'Estimate severity'
+teaching: 10
+exercises: 2
+editor_options: 
+  chunk_output_type: inline
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- Why do we estimate the clinical severity of an epidemic?
+
+- How can the Case Fatality Risk (CFR) be estimated early in an ongoing epidemic?
+
+- How do we estimate other severity measurements from different data sources?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Estimate the CFR from aggregated case data using `{cfr}`.
+
+- Estimate a delay-adjusted CFR using `{epiparameter}` and `{cfr}`.
+
+- Estimate severity measurements like the Hospitalization Fatality Risk (HFR) using `{cfr}`.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: prereq
+
+## Prerequisites
+
+This episode requires you to be familiar with:
+
+**Data science** : Basic programming with R.
+
+**Epidemic theory** : Epidemiological parameters. Time periods.
```

Comment:
I agree, I inherited the term from another episode introducing delay distributions.

```suggestion
**Epidemic theory** : Epidemiological parameters and Delay distributions.
```